scripts/utils/status-badge.js

```diff
@@ -0,0 +1,4 @@
+export default function createStatusBadge(workflowFile, stylelintVersion) {
+	// TODO : rename `romainmenke` to `stylelint`
+	return `[![${stylelintVersion}](https://github.com/romainmenke/stylelint-ecosystem-tester/actions/workflows/${workflowFile}/badge.svg)](https://github.com/romainmenke/stylelint-ecosystem-tester/actions/workflows/${workflowFile})`;
```

Comment:
```suggestion
	return `[![${stylelintVersion}](https://github.com/stylelint/stylelint-ecosystem-tester/actions/workflows/${workflowFile}/badge.svg)](https://github.com/stylelint/stylelint-ecosystem-tester/actions/workflows/${workflowFile})`;
```

Also update the files after applying this change.

Reply:
Let's update!